packages/jetpack-blocks/package.json

```diff
@@ -8,49 +8,53 @@
 	],
 	"dependencies": {
 		"@automattic/format-currency": "file:../format-currency",
-		"@babel/polyfill": "^7.2.5",
-		"@wordpress/api-fetch": "^2.2.8",
-		"@wordpress/blob": "^2.1.0",
-		"@wordpress/blocks": "^6.0.7",
-		"@wordpress/components": "^7.0.8",
-		"@wordpress/compose": "^3.0.1",
-		"@wordpress/data": "^4.2.1",
-		"@wordpress/date": "^3.0.1",
-		"@wordpress/edit-post": "^3.1.11",
-		"@wordpress/editor": "^9.0.11",
-		"@wordpress/element": "^2.1.9",
-		"@wordpress/hooks": "^2.0.5",
-		"@wordpress/i18n": "^3.1.1",
-		"@wordpress/keycodes": "^2.0.6",
-		"@wordpress/plugins": "^2.0.11",
-		"@wordpress/token-list": "^1.1.0",
-		"@wordpress/url": "^2.3.3",
-		"classnames": "^2.2.6",
-		"cookie": "^0.3.1",
-		"email-validator": "^2.0.4",
-		"gridicons": "^3.1.1",
-		"lodash": "^4.17.11",
+		"@babel/polyfill": "7.2.5",
+		"@wordpress/api-fetch": "2.2.8",
+		"@wordpress/blob": "2.1.0",
+		"@wordpress/blocks": "6.0.7",
+		"@wordpress/components": "7.0.8",
+		"@wordpress/compose": "3.0.1",
+		"@wordpress/data": "4.2.1",
+		"@wordpress/date": "3.0.1",
+		"@wordpress/edit-post": "3.1.11",
+		"@wordpress/editor": "9.0.11",
+		"@wordpress/element": "2.1.9",
+		"@wordpress/hooks": "2.0.5",
+		"@wordpress/i18n": "3.1.1",
+		"@wordpress/keycodes": "2.0.6",
+		"@wordpress/plugins": "2.0.11",
+		"@wordpress/token-list": "1.1.0",
+		"@wordpress/url": "2.3.3",
+		"classnames": "2.2.6",
+		"cookie": "0.3.1",
+		"duplicate-package-checker-webpack-plugin": "3.0.0",
+		"email-validator": "2.0.4",
+		"gridicons": "3.1.1",
+		"lodash": "4.17.11",
 		"markdown-it": "8.4.2",
-		"photon": "^2.0.1",
-		"refx": "^3.1.1",
-		"resize-observer-polyfill": "^1.5.1",
+		"photon": "2.0.1",
+		"refx": "3.1.1",
+		"resize-observer-polyfill": "1.5.1",
 		"swiper": "4.4.6"
 	},
 	"devDependencies": {
-		"chalk": "^2.4.2",
-		"copy-webpack-plugin": "^4.6.0",
-		"enzyme": "^3.9.0",
-		"enzyme-adapter-react-16": "^1.9.1",
-		"enzyme-to-json": "^3.3.5",
-		"react": "^16.8.3",
-		"webpack": "^4.29.6"
+		"@automattic/calypso-build": "file:../calypso-build",
+		"@automattic/calypso-color-schemes": "file:../calypso-color-schemes",
+		"chalk": "2.4.2",
```

Comment:
I can't see where `chalk` is used?

Reply:
It was in `bin/build.js`
```suggestion
```

Reply:
Removed. Will need to re package-lock (unless we remove the whole file https://github.com/Automattic/wp-calypso/pull/31503#issuecomment-474333824).